README.md

```diff
@@ -471,6 +509,20 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 See https://staticsitegenerators.net and https://www.staticgen.com
 
+
+## Ticketing systems
+*Web-based ticketing system.*
```

Comment:
Could we move this to the existing Task and Bug trackers subsection? We might need to edit the wording (include Ticketing)